main.go

```diff
@@ -33,6 +33,7 @@ var (
 	// This variable is set at build time using -ldflags parameters.
 	// See: https://stackoverflow.com/a/11355611
 	gitSha1 string
+	options *compileopts.Options
```

Comment:
Globals are a bad idea. Can you pass this options object manually to all `executeCommand` calls? You can usually use `config.Options` for that.

Reply:
You are right, globals should be avoided....
But I thought it was tolerated when data is  initialized once and reused multiple times (which is the case of config object ?)

However, I'll follow your advice and remove this global .